modules/compliance-kubeletconfig-sub-pool-remediation.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-kubeletconfig-sub-pool-remediation"]
+= Remediating `KubeletConfig` sub pools
+
+* If you want to apply `KubeletConfig` remediation on `MachineConfigPool` sub-pools, add a label to the sub-pool `MachineConfigPool` CR:
```

Comment:
Consider making this module follow the [procedure](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc) template.